packages/eslint-config/package.json

```
@@ -12,7 +12,7 @@
   "peerDependencies": {
     "@typescript-eslint/eslint-plugin": "^1.11.0",
     "@typescript-eslint/parser": "^1.11.0",
-    "eslint": "^5.16.0",
+    "eslint": "^6.0.0",
```

I feel that bumping up this peer-dependency range is a breaking change, because it forces our users to upgrade to eslint v6 too and thus bring in new recommended rules.

As a backwards-compatible alternative, perhaps we can keep supporting eslint v5?

```suggestion
    "eslint": "^5.16.0 || ^6.0.0",
```

My only concern is that we have no easy way how to verify that our rules actually work with eslint v5 and that we are not relying on any new rules/something else added in v6 only.

---

As long as we release `@loopback/eslint-config` as a new major, we can have `^6.0.0` only.